### PR TITLE
fix: Document limitations of Postgres CDC syncing

### PR DESCRIPTION
**Pull request by bbernays**

### CODE REVIEW

File: website/pages/docs/plugins/sources/postgresql/cdc.md [hunk 1]
```diff
@@ -61,3 +61,11 @@ You will need to set the `cloudsql.logical_deconging` configuration to on. See f
...
 
 Please follow the Official Azure documentation on [how to enable logical replication](https://learn.microsoft.com/en-us/azure/postgresql/single-server/concepts-logical) via the CLI or Console (UI).
+
+
+## Limitations:
+
+The following event types are not supported:
+- DDL changes (e.g. `CREATE TABLE`, `ALTER TABLE`, etc)
+- `TRUNCATE` statements (e.g. `TRUNCATE TABLE table_name`)
+- `Delete` statements (e.g. `DELETE FROM table_name`)
```
Comment (candiduslynx) [inline website/pages/docs/plugins/sources/postgresql/cdc.md:71]:
```suggestion
- `DELETE` statements (e.g. `DELETE FROM table_name`)
```

File: plugins/source/postgresql/client/cdc.go [hunk 2]
```diff
@@ -226,31 +226,6 @@ func (c *Client) listenCDC(ctx context.Context, res chan<- message.SyncMessage)
 				}
 				res <- resource
 			case *pglogrepl.DeleteMessage:
-				rel, ok := relations[logicalMsg.RelationID]
```
Comment (candiduslynx) [inline plugins/source/postgresql/client/cdc.go:229]: Is there an issue why we're removing this?

Comment (bbernays) [reply plugins/source/postgresql/client/cdc.go:229]: Because it doesn't do anything

File: website/pages/docs/plugins/sources/postgresql/cdc.md [hunk 3]
```diff
@@ -61,3 +61,11 @@ You will need to set the `cloudsql.logical_deconging` configuration to on. See f
 ## Azure DB (PostgreSQL)
 
 Please follow the Official Azure documentation on [how to enable logical replication](https://learn.microsoft.com/en-us/azure/postgresql/single-server/concepts-logical) via the CLI or Console (UI).
+
+
+## Limitations:
```
Comment (hermanschaaf) [inline website/pages/docs/plugins/sources/postgresql/cdc.md:66]:
```suggestion
## Limitations
```

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**